Amethyst/Managers/WindowManager.swift

```diff
@@ -670,6 +670,13 @@ extension WindowManager: WindowTransitionTarget {
         return windows.isWindowFloating(window)
     }
 
+    func getCurrentLayout() -> Layout<Application.Window>? {
+        guard let screenManager: ScreenManager<WindowManager<Application>> = focusedScreenManager(), let layout = screenManager.currentLayout else {
```

Comment:
Optional chaining might simplify this since we're just returning the value.

Reply:
Thanks, fixed.